device/accton/x86_64-accton_as4630_54te-r0/sonic_platform/component.py

```diff
@@ -0,0 +1,121 @@
+#############################################################################
+# Celestica
```

Comment:
Celestica? Copy/paste issue?

Reply:
Yes, it is copy/paste issue. I'll remove the keyword.